dietpi/dietpi-software

```diff
@@ -12814,11 +12816,9 @@ If no WireGuard (auto)start is included, but you require it, please do the follo
 		if To_Uninstall 141 # ADS-B Feeder
 		then
 			G_EXEC_NOHALT=1 G_EXEC_OUTPUT=1 G_EXEC /opt/adsb/docker-compose-adsb down
-			Remove_Service adsb-bootstrap
+			Remove_Service adsb-docker
 			Remove_Service adsb-feeder-update
 			Remove_Service adsb-setup
-			Remove_Service adsb-update
-			command -v docker &> /dev/null && G_EXEC docker image prune -a -f
```

Comment:
Does this mean the image may remain on the system? Probably we can search for a matching container and image, if they have a consistent name pattern, like we do on Portainer uninstall:
```sh
			# Check if Docker is still installed
			if [[ -d '/mnt/dietpi_userdata/docker-data' ]]
			then
				# Remove ADS-B Feeder container & image
				local container=$(docker container ls -a | mawk '/adsb-feeder( |$)/{print $1;exit}')
				[[ $container ]] && G_EXEC docker container rm -f "$container"
				local image=$(docker image ls -a | mawk '/adsb-feeder( |$)/{print $3;exit}')
				[[ $image ]] && G_EXEC docker image rm "$image"
			fi
```

Reply:
I tried to figure out if that was worth doing and ended up giving up. The names of the images are... fairly inconsistent. I could add a list of them here, but that seemed very noisy and very specific.
I think it would make more sense to do this upstream in the adsb-feeder-image repo and have a helper function that could be called here. I'll have to play with that.

Reply:
Ah right, makes sense with the large number of containers. Images can be given a tag right? Probably that can be used to make them identifiable. Or otherwise some kind of state file?